editor/editor_export.cpp

```diff
@@ -1632,15 +1632,17 @@ void EditorExportPlatformPC::get_preset_features(const Ref<EditorExportPreset> &
 }
 
 void EditorExportPlatformPC::get_export_options(List<ExportOption> *r_options) {
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/debug", PROPERTY_HINT_GLOBAL_FILE), ""));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, "custom_template/release", PROPERTY_HINT_GLOBAL_FILE), ""));
+
+	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "binary_format/64_bits"), true));
+	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "binary_format/embed_pck"), false));
+
 	r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "texture_format/bptc"), false));
```

Comment:
BTW @reduz @Calinou is BPTC still used in `3.2` / `4.0`?

We still have some code that references it but I don't see any platform preset that actively enables it. And there's no code using this `texture_format/bptc` option at all.

Reply:
BPTC is still available in 3.2, but is disabled by default as it can only be used on Windows and Linux with relatively recent desktop GPUs. (I think it's only available when using GLES3 too.)

I don't know about `master` though.

Reply:
BPTC is supported, but I think `EditorExportPlatformPC::get_preset_features()` is lacking a check for `texture_format/bptc`, since it's not assumed to be an always available PC feature.

Reply:
I mean, I'm a bit puzzled because, considering `EditorExportPlatformPC::get_platform_features()` doesn't add `bptc` to the list and the export preset setting is not used, how can you currently get BPTC textures in your export? I'm checking all this in 3.2.

Reply:
Maybe @elasota can clarify?

Reply:
I'm not 100% sure of what's going on with BPTC and it looks like the behavior is different in master vs. 3.2 too.

I think `texture_format/bptc` and `texture_format/no_bptc_fallbacks` are obsolete because the fallback system was removed, BPTC was rolled into the S3TC feature, and the choice of whether to enable BPTC became a project option, but that also means that "S3TC" in the export options is a misnomer because it controls both formats.

AFAIK as of 0565adb20ff01a758c5ae8e4534a2cb7f07952df (and then fixed by pull 21256), BPTC was compacted into the S3TC option and changed to use the `rendering/vram_compression/import_bptc` project setting to control whether it's enabled at all.

When the project option is enabled, then `compress/bptc_ldr` on the texture import settings also affects whether or not it's used for RGB textures.